README.md

```diff
@@ -1,4 +1,4 @@
-# Remote-friendly companies
+    # Remote-friendly companies
```

Comment:
This change to the readme needs to be undone. Other than that, the PR looks good.

Reply:
Fixed